### PR TITLE
Data table format provider docs

### PR DESCRIPTION
**Pull request by DKaramfilov**

### CODE REVIEW

File: libraries/radspreadprocessing/formats-and-conversion/data-table/data-table-format-provider.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title:  DataTableFormatProvider
+page_title:  DataTableFormatProvider
+description:  DataTableFormatProvider
```
Comment (TanyaDimitrova) [inline libraries/radspreadprocessing/formats-and-conversion/data-table/data-table-format-provider.md:4]: Add meaningful description

File: libraries/radspreadprocessing/formats-and-conversion/data-table/settings.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+---
+title: Settings
+page_title: Settings
+description: This article shows the seeting taht allow you to control the dataTable import/export operations.
```
Comment (TanyaDimitrova) [inline libraries/radspreadprocessing/formats-and-conversion/data-table/settings.md:4]: This article shows the settings that you can use to control the DataTable import/export operations.

File: libraries/radspreadprocessing/formats-and-conversion/data-table/settings.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+---
+
+
+# Import Settings
+
+* __PreferredDateTimeFormat:__ This property gets or sets the default format string when importing DateTime columns.
+* __ShouldImportColumnHeaders:__ Controls whether the column headers should be imported. 
+* __StartCellIndex:__ Get or sets the index where the table should start in the Worksheet.
+
+## The OnCellImported event
```
Comment (TanyaDimitrova) [inline libraries/radspreadprocessing/formats-and-conversion/data-table/settings.md:18]: CellImported

**Review by TanyaDimitrova — CHANGES_REQUESTED** [review]
The Overview topics also need to be updated. The new assembly should be added to the Getting Started topic as well. Also, verify that all other documentations that list the assemblies are updated (e.g. [AJAX](https://docs.telerik.com/devtools/aspnet-ajax/installation/included-assemblies#telerik-document-processing-libraries), [Blazor](https://docs.telerik.com/blazor-ui/components/document-processing/overview) )

File: libraries/radspreadprocessing/formats-and-conversion/data-table/data-table-format-provider.md (hunk 1 above)
Comment (DKaramfilov) [reply libraries/radspreadprocessing/formats-and-conversion/data-table/data-table-format-provider.md:4]: removed

File: libraries/radspreadprocessing/formats-and-conversion/data-table/settings.md (hunk 2 above)
Comment (DKaramfilov) [reply libraries/radspreadprocessing/formats-and-conversion/data-table/settings.md:4]: fixed

File: libraries/radspreadprocessing/formats-and-conversion/data-table/settings.md (hunk 3 above)
Comment (DKaramfilov) [reply libraries/radspreadprocessing/formats-and-conversion/data-table/settings.md:18]: fixed

File: libraries/radspreadprocessing/formats-and-conversion/data-table/settings.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+The __OnCellImported__  event is fired for each cell and allows you to change the cell properties.
```
[the remark this answers is not in the document]
Comment (DKaramfilov) [reply libraries/radspreadprocessing/formats-and-conversion/data-table/settings.md:20]: fixed

**Pull request merged**